src/digitalmarketplace/all.js

```diff
@@ -12,6 +13,10 @@ function initAll (options) {
   if ($cookieSettingsPage) {
     new CookieSettings($cookieSettingsPage).init()
   }
+  const currentConsentCookie = getConsentCookie()
+  if (currentConsentCookie && currentConsentCookie.analytics) {
+    initAnalytics()
```

Comment:
Remind me, if a user changes their cookie settings and accepts analytics, will this trigger?

Reply:
That has its own trigger, which is why the general case got missed.

Reply:
cool cool! 👍